metapool/data/abs_quant_sample_info_calc.yml

```diff
@@ -2,8 +2,10 @@
   "zymo_dna_rna_shield":
     "storage_liquid_lot_number":
       "219839":
+        "mass_storage_tube_and_storage_liquid_before_sample_g": 16.9
```

Comment:
... just to confirm, where are they coming from? Any other columns not needed any longer?

Reply:
Going forward, the Mayo is going to weigh every tube individually and provide that value in a "mass_storage_tube_and_storage_liquid_before_sample_g" column in the metadata they send us (instead of us adding that column and filling it in based on the lot average).

Reply:
Thank you. Just to confirm, should we assume that this will be part of the sample or the prep-info file? and what should be the role of the SPP here?

Reply:
Thank you for the catch @mmbryant23 ! @antgonza they were most likely recently removed and I merged them back in by accident from Caitriona's branch. Things recently removed from the master can look like new lines of code/text in the feature branch. I'll remove them.